styleguide/source/assets/scss/02-molecules/_article-stub.scss

```diff
@@ -0,0 +1,71 @@
+.ama__article-stub {
+  @include gutter($margin-bottom-half...);
+
+  &--container-image {
```

Comment:
Is this a sub element or a modifier? Seems like it is probably a sub element.

Reply:
good quesiton

Reply:
i think it's neither a modifer or sub element. but just a name